enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCollectionModalFilters.tsx

```diff
@@ -0,0 +1,58 @@
+import { c, t } from "ttag";
+import _ from "underscore";
+
+import { Icon, Select, Flex, Text, Switch } from "metabase/ui";
+
+import { dateFilterOptions, isDateFilter, type DateFilter } from "./utils";
+
+interface CleanupCollectionModalFiltersProps {
+  dateFilter: DateFilter;
+  recursiveFilter: boolean;
+  onDateFilterChange: (dateFilter: DateFilter) => void;
+  onRecursiveFilterChange: (recursiveFilter: boolean) => void;
+}
+
+export const CleanupCollectionModalFilters = ({
+  dateFilter,
+  recursiveFilter,
+  onDateFilterChange,
+  onRecursiveFilterChange,
+}: CleanupCollectionModalFiltersProps) => (
+  <Flex
+    mt="1.5rem"
+    mb="1rem"
+    w="100%"
+    justify="space-between"
+    direction={{ base: "column", md: "row" }}
+    align={{ base: "start", md: "center" }}
+    gap={{ base: "md", md: "none" }}
+  >
+    <Flex
+      gap="sm"
+      align={{ base: "start", md: "center" }}
+      direction={{ base: "column", md: "row" }}
+    >
+      <Text fw="bold">{c(
+        `Prefixes a time span, reads as "Not viewed in over 6 months"`,
+      ).t`Not viewed in over`}</Text>
```

Comment:
Could `ttag`'s `jt` function work here, with the `<Select />` placed into the JSX expression? Then this can localize in languages where the duration would be placed at the start or in the middle of the phrase, like German, Japanese, Chinese, and Korean. (A little bird named ChatGPT told me.)

For example in German it would be: "Seit mehr als 6 Monaten nicht gesehen." It would be nice to put the dropdown in the middle of that sentence, if possible.

Reply:
I'm going to bring the us to product. Might make an issue and merge for now if that's okay?

Reply:
Fixed